src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Commands.xml

```diff
@@ -3,49 +3,82 @@
      Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. -->
 <Commands>
   <Verb Name="add">
-    <Noun Name="source">
+    <Noun Name="source" Version="3.1.200"
+       Description="adds a new package source to your NuGet configuration files."
+       >
       <Argument Name="Source" LongName="PackageSourcePath" Help="SourcesCommandSourceDescription" />
       <SingleValueOption Name="name" Shortcut="n" Help="SourcesCommandNameDescription"/>
-      <SingleValueOption Name="username" Shortcut="u" Help="SourcesCommandUserNameDescription"/>
+      <SingleValueOption Name="username" Shortcut="u" Help="SourcesCommandUsernameDescription"/>
       <SingleValueOption Name="password" Shortcut="p" Help="SourcesCommandPasswordDescription"/>
       <SwitchOption Name="store-Password-In-Clear-Text" Help="SourcesCommandStorePasswordInClearTextDescription"/>
       <SingleValueOption Name="valid-Authentication-Types" Help="SourcesCommandValidAuthenticationTypesDescription" />
       <SingleValueOption Name="configfile" Help="Option_ConfigFile" />
+      <Example Title="Add `nuget.org` as a source:" Command="dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org" />
+      <Example Title="Add `c:\packages` as a local source:" Command="dotnet nuget add source c:\packages" />
+      <Example Title="Add a source that needs authentication:" Command="dotnet nuget add source https://someServer/myTeam -n myTeam -u myUsername -p myPassword --store-password-in-clear-text" />
+      <Example Title="Add a source that needs authentication (then go install credential provider):" Command="dotnet nuget add source https://azureartifacts.microsoft.com/myTeam -n myTeam" />
```

Comment:
We should probably use contoso for the example urls.

Reply:
which one would you change? the ones with myTeam? can you make the suggesting more specific.

- nuget.org 
- c:\packages
- https://someserver/myTeam
- https://azureartifacts.microsoft.com/myTeam

Reply:
azureartifacts.microsoft.com to contoso. 

Basically the base part of each url should be contoso.

Reply:
i was hoping to make that a real azure artifacts example. perhaps i need to tweak it to be more real.

Reply:
I was under the impression that all examples in docs should be contoso to avoid confusion.

Reply:
I'm asking for feedback from the sdk lead who reviewed these docs:  https://github.com/dotnet/docs/pull/17401#issuecomment-606770645

Reply:
added more contoso in latest commit based on your feedback and then discussion with jonathan & tom (docs lead)